jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/colibri/Exceptions.kt

```diff
@@ -18,6 +18,8 @@ package org.jitsi.jicofo.bridge.colibri
 /** Bridge selection failed, i.e. there were no bridges available. */
 class BridgeSelectionFailedException : Exception("Bridge selection failed")
 
+class ConferenceAlreadyExistsException : Exception()
```

Comment:
Should this extend `ColibriAllocationFailedException`?

Reply:
Right, make it as extend  for ColibriAllocationFailedException